### ISSUE TITLE
RE8 - Pitch lock

### ISSUE THREAD

**Issue by MrOmbre** [praydog/REFramework#291]
If I deactivate Pitch in REFramework Menu, I can use right thumpstick (Y axis) to move both hands, please can you lock this completly?

**Comment by praydog** [comment]
Will be fixed in a later update for 7/8.

**Comment by MrOmbre** [comment]
nice to know.
will there be a changelog btw. the versions?

**Comment by MrOmbre** [comment]
> Will be fixed in a later update for 7/8.

Hi, maybe I missunderstood this, but did you fix it in a private build?

**Comment by praydog** [comment]
Yes. I fixed it a while ago. All of the fixes are planned to be released in one large update.

**Comment by praydog** [comment]
Fixed in #319
